brands/amenity/cafe.json

```diff
@@ -622,6 +633,17 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Sharetea": {
+    "countryCodes": ["au"],
```

Comment:
Sharetea is also found in Taiwan, the U.S., and other countries. However, a separate entry would have to be added for Chinese-speaking countries, because the English name would be secondary to the Chinese name.

Reply:
>  English name would be secondary to the Chinese name.

Yep, that's why I didn't include `tw` in the countryCodes, since it will probably need to be a separate entry.

I've updated to include US.